sample-java-model/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id "java"
+}
```

Comment:
Will do.  I had the new plugins form in place originally since I though I was going to need the apt plugin, but alas, compileOnly is natively supported now!  w00t